woocommerce/woocommerce-gateway-stripe
# Fix tooltip positioning on settings page for small screens

# Changes proposed in this Pull Request:

Fixes #2045

When the UPE redesign settings page is active, and the user has a small screen, the tooltip can get cut off because of the center positioning.

This PR position the tooltip on the top left if the center positioning will cause it to go off-screen.

# Testing instructions

1. Enable UPE and the UPE redesign on http://localhost:8082/wp-admin/admin.php?page=wc-stripe-dev
2. Emulate a mobile device using the browser dev tools, or use a small device to test.
3. The tooltip should be positioned like the screenshot below.
4. The positioning of the tooltip on bigger screens shouldn't change.

## Before

![image](https://user-images.githubusercontent.com/4153905/138503917-52c155a0-3fd0-4ea5-a4ca-18f84c08b005.png)

## Expected result

<img width="410" alt="image" src="https://user-images.githubusercontent.com/4153905/138503766-1fe86795-5d0c-46a5-9312-9a940cd634eb.png">
